### PR TITLE
New BlogVault migration doc

### LINKED ISSUE #1200: Recommend blogvault for all WordPress imports
Pending more testing. 
Create new: 
- [ ] /sites/migrate/bv-pantheon-plugin
 - [ ] Outline here

add links from: 
- [ ] https://pantheon.io/docs/articles/sites/migrate/export-an-existing-wordpress-site/
- [ ] /sites/migrate
- [ ] /sites/migrate/manual

### PR DESCRIPTION
**Pull request by nataliejeremy**
Closes #1200 
@rachelwhitton Please review.

### CODE REVIEW

File: source/docs/articles/sites/migrate/bv-pantheon-plugin.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+1. [Create a Pantheon account](https://dashboard.pantheon.io/register) or log in to your account.
+2. Create and install a new WordPress site on Pantheon:
+ - Choose **Create a new site** from your Pantheon Dashboard.
+ - Name your site.
+ - Select **Start from Scratch**, and choose **WordPress**.
+ - Click **Visit Development Site** and complete the installation process.
+3. Save your SFTP connection information.
+![SFTP Connection Information](/source/docs/assets/images/sftp-connection.png)
+4. Right click on the **Dev tab** and select **Copy Link Address**.
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/bv-pantheon-plugin.md]: Right-clicking should be on "visit development site" not the Dev Tab.

**Comment by nataliejeremy** [comment]
Looping in @ataylorme to review

**Comment by rachelwhitton** [comment]
+1

**Comment by ataylorme** [comment]
@nataliejeremy 

> The WordPress Native PHP Sessions plugin is automatically installed during the migration process. For more details on this plugin, see WordPress and PHP Sessions.

I've reached out to BlogVault for some clarification around this. I want to make sure the plugin is installed _and_ activated. If it's just installed, but not activated, before a codebase using PHP Sessions is migrated over there will still be issues.

On both the WordPress and WP Engine migration pages I think it would be good to link to our [platform considerations doc](https://pantheon.io/docs/articles/sites/platform-considerations/). Most people migrating sites will be new to the platform and that can be a helpful doc as it highlights items like PHP short tags, which they may need to consider before migrating.

Otherwise this looks really solid - nice job!

**Comment by rachelwhitton** [comment]
@nataliejeremy I revisited this one today and made some follow up edits, can you review? 
- [ ] Redirect Required: [/docs/wordpress-duplicator](https://pantheon.io/docs/wordpress-duplicator) >  [/docs/migrate-wordpress-duplicator](https://pantheon.io/docs/migrate-wordpress-duplicator)


@ataylorme any response from bv on the WordPress Native PHP Sessions plugin? Let me know if I can help out with any testing to get this one shipped. 

+1

**Comment by ataylorme** [comment]
@rachelwhitton We actually heard from BlogVault today that the WordPress Native PHP Sessions plugin integration shipped.

They are working on Terminus integration, using machine tokens instead of SFTP credentials, as well.

Perhaps @joshkoenig can comment on whether we want to wait for that change or not since the doc will also need to change to reflect the new process.

**Comment by rachelwhitton** [comment]
Nice! Thanks for the update @ataylorme 
My preference would be to track updates for BlogVault's Terminus integration within a separate issue so this one isn't blocked.
cc @joshkoenig @nataliejeremy

**Comment by joshkoenig** [comment]
LGTM. We can revise in April.

**Pull request merged**